packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -1106,4 +1108,14 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
       </Label>
     ) : null;
   };
+
+  /**
+   * Returns true if dropdown should set to open on focus.
+   * Otherwise, isOpen state should be toggled on click
```

Comment:
Suggestion:

```ts
  /**
   * Determines if the Dropdown should be open on focus or not.
   */
```

Reply:
I specifically want to callout "Otherwise, isOpen state should be toggled on click" because the method is used in onClick handler, which can be quite confusing

Reply:
Ah. i see your suggestion on moving the other comment inlined where the method is used. i like you suggestion better than my version. i will update these in my future PR :)